static/translations/messages_en.json

```diff
@@ -208,6 +208,7 @@
   "Reactions": "Reactions",
   "No reactions": "No reactions",
   "See who reacted": "See who reacted",
+  "Muted User": "Muted User",
```

Comment:
The webapp uses sentence case ("Muted user") for all translatable strings, so we should probably do that here for consistency.